### ISSUE TITLE
setup:config creates an unwritable ~/.acquiacli directory.

### ISSUE THREAD

**Issue by jlfranklin** [typhonius/acquia_cli#134]
**Describe the bug**
When the user first configures their account with `setup:config`, the process fails because acquiacli creates a directory it cannot write to.

**To Reproduce**

```
franklin@llama:~$ rm -rf ~/.acquiacli/
franklin@llama:~$ ./acquiacli.phar setup:config
➜                                                                  
➜   Global configuration (/home/franklin/.acquiacli/acquiacli.yml) 
➜                                                                  
?  Global configuration file not found. Would you like to add one? (y/n) y
?  What is your Acquia key? [redacted]
?  What is your Acquia secret? 
➜                                                            
➜   /home/franklin/.acquiacli/acquiacli.yml is not writeable 
➜                                                            
franklin@llama:~$ ls -al .acquiacli/
ls: cannot open directory '.acquiacli/': Permission denied
franklin@llama:~$ ls -ald .acquiacli
d-w-r-xr-T 2 franklin franklin 4096 Nov 20 10:46 .acquiacli
franklin@llama:~$ php -v
PHP 7.3.19-1~deb10u1 (cli) (built: Jul  5 2020 06:46:45) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.3.19, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.3.19-1~deb10u1, Copyright (c) 1999-2018, by Zend Technologies
franklin@llama:~$ lsb_release -a
No LSB modules are available.
Distributor ID:	Debian
Description:	Debian GNU/Linux 10 (buster)
Release:	10
Codename:	buster
franklin@llama:~$ ./acquiacli.phar --version
AcquiaCli 2.0.9
```

**Expected behavior**
A new $HOME/.acquiacli/acquiacli.yml with the API key.

**Desktop (please complete the following information):**
 - Debian 10
 - PHP 7.3.19
 - AcquiaCLI 2.0.9

**Comment by typhonius** [comment]
I have a feeling that this is the same issue that I found and fixed in #133. I'll push out a new tag later today which should allow you to use the `self:update` command and have this fixed.

**Comment by typhonius** [comment]
This has now been pushed out as part of [2.0.10](https://github.com/typhonius/acquia_cli/releases/tag/2.0.10) so you should be able to `self:update` to make it all work. Thanks for reporting!
